boards/sim/sim/sim/scripts/Make.defs

```diff
@@ -296,9 +296,9 @@ ifeq ($(CONFIG_HOST_MACOS),y)
 else
   LDFLAGS += -Wl,--gc-sections
 
-  # Let the symbol table link from 0x50000000
+  # Let the symbol table link from 0x30000000
```

Comment:
Should update together
https://github.com/apache/nuttx/blob/master/arch/sim/src/cmake/Toolchain.cmake#L196

Reply:
ok